Tasks/DownloadBuildArtifactsV0/task.loc.json

```diff
@@ -10,7 +10,7 @@
   "version": {
     "Major": 0,
     "Minor": 217,
```

Comment:
Bump should be `218` and patch to `0`.

Please see [instructions](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md). 

`Since there is cut off on Tuesday of the 3rd week of the sprint - for changes on 3rd sprint week after Tuesday - set it up as (current sprint number) + 1. For this case changes will be shipped with the next release.`

Reply:
Manual courtesy push will be run, it is ok to patch in that case.